### ISSUE TITLE
Retraction Watch database

### ISSUE THREAD

**Issue by gkaramanis** [rfordatascience/tidytuesday#613]
The Retraction Watch database has been acquired by Crossref and become public.

Details:
https://www.crossref.org/blog/news-crossref-and-retraction-watch/

The dataset can be directly downloaded or read from [https://api.labs.crossref.org/data/retractionwatch](https://api.labs.crossref.org/data/retractionwatch) (CSV)

**Comment by jmcastagnetto** [comment]
Just a small correction (I think), Retraction Watch and Crossref have entered a 5-year partnership agreement, so it does not seem like a business acquisition, according to 

"Scientific retractions may become easier to spot as Retraction Watch finds new partner"
*Five-year deal will share journal watchdog’s widely used database with Crossref in return for stable funding*
https://www.science.org/content/article/scientific-retractions-may-become-easier-spot-retraction-watch-finds-new-partner

**Comment by gkaramanis** [comment]
Correct, the acquisition is about the database only. Crossref says “the Retraction Watch database has been acquired by Crossref and made a public resource.”

**Comment by jonthegeek** [comment]
Currently gives me a 503 (Service Temporarily Unavailable).

**Comment by gkaramanis** [comment]
I could download it before I got the 503 😃 

[retractions.csv](https://github.com/rfordatascience/tidytuesday/files/15294140/retractions.csv)

**Comment by lgibson7** [comment]
Hi @gkaramanis. Thanks for submitting this issue. Would you be willing to submit the data set through a PR? You can find the instructions on how to do so [here](https://github.com/rfordatascience/tidytuesday/blob/master/data/curated/template/instructions.md).
